### PR TITLE
Milena Filipovic assignment1

### PR DESCRIPTION
**Pull request by MilenaFilipovic**

### CODE REVIEW

File: MilenaFilipovic/assignment-1/anagrams_test.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import unittest
+
+from anagrams import are_anagrams
+
+
+class TestAnagrams(unittest.TestCase):
+
+  def test_both_empty_or_none(self):
+    self.assertEqual(are_anagrams('', ''), False, 'Empty strings - False')
```
Comment (ariava) [inline MilenaFilipovic/assignment-1/anagrams_test.py]: I think you can directly use assertTrue and assertFalse (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertTrue), they will make this kind of tests more readable.

File: MilenaFilipovic/assignment-1/main.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+"""Example usage of linkedlist.py and klast.py."""
```
Comment (ariava) [inline MilenaFilipovic/assignment-1/main.py]: Change the names of the files in this comment.

File: MilenaFilipovic/assignment-1/main.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+from anagrams_sentence import are_sentence_anagrams
+from anagrams_sentence_words import are_word_sentence_anagrams
+
+
+def main(args):
+  if len(args) < 3:
+    print('Incorrect input.')
+    return
+
+  if args[0] == 'og':
```
Comment (ariava) [inline MilenaFilipovic/assignment-1/main.py]: This is cool, you can also try to use argparse (https://docs.python.org/3/library/argparse.html), which is a python library that should make it easier to handle command line arguments.

File: MilenaFilipovic/assignment-1/anagrams_test.py (hunk 1 above)
Comment (MilenaFilipovic) [reply MilenaFilipovic/assignment-1/anagrams_test.py]: Thank you - I'll definitely use this next time!

File: MilenaFilipovic/assignment-1/main.py (hunk 2 above)
Comment (MilenaFilipovic) [reply MilenaFilipovic/assignment-1/main.py]: I'll make sure to keep an eye on these kinds of errors.

File: MilenaFilipovic/assignment-1/main.py (hunk 3 above)
Comment (MilenaFilipovic) [reply MilenaFilipovic/assignment-1/main.py]: Thanks! I'll look into it and use it for the subsequent assignments!

File: MilenaFilipovic/assignment-1/main.py (hunk 2 above)
Comment (ariava) [reply MilenaFilipovic/assignment-1/main.py]: You can change the comment and commit the change to this branch and the pull request will be automatically updated :)

**Pull request merged**